pkg/ccl/backupccl/testdata/backup-restore/in-progress-imports

```diff
@@ -0,0 +1,469 @@
+----
+
+
+exec-sql
+BACKUP DATABASE d INTO 'nodelocal://0/database_upgrade/' with revision_history;
+----
+
+
+save-cluster-ts tag=m0
+----
+
+
+# Conduct another set of incremental backups to ensure that _offline_ tables do _not_ fully backup
+# in a cluster in an unfinalized state
+exec-sql
+BACKUP INTO LATEST IN 'nodelocal://0/cluster/' with revision_history;
+----
+
+
+exec-sql
+BACKUP DATABASE d INTO LATEST IN 'nodelocal://0/database/' with revision_history;
+----
+
+
+exec-sql
+CREATE VIEW show_cluster_backup AS
+SELECT
+  database_name, object_name, object_type, rows, backup_type
+FROM
+  [SHOW BACKUP FROM LATEST IN 'nodelocal://0/cluster/']
+WHERE
+  object_name = 'foo' or object_name = 'foofoo';
+----
+
+
+exec-sql
+CREATE VIEW show_database_backup AS
+SELECT
+  database_name, object_name, object_type, rows, backup_type
+FROM
+  [SHOW BACKUP FROM LATEST IN 'nodelocal://0/database/']
+WHERE
+  object_name = 'foo' or object_name = 'foofoo';
+----
+
+
+query-sql
+SELECT * FROM show_cluster_backup;
+----
+d foo table 1 full
+d foofoo table 2 full
+d foo table 0 incremental
+d foofoo table 0 incremental
+
+
+query-sql
+SELECT * FROM show_database_backup;
+----
+d foo table 1 full
+d foofoo table 2 full
+d foo table 0 incremental
+d foofoo table 0 incremental
+
+
+exec-sql
+SET CLUSTER SETTING jobs.debug.pausepoints = '';
+----
+
+
+# Resume the job so the next set of incremental backups observes that tables are back online
+job resume=b
+----
+
+
+job tag=b wait-for-state=succeeded
+----
+
+
+job resume=bb
+----
+
+
+job tag=bb wait-for-state=succeeded
+----
+
+
+# Key Test: Ensure that once the tables come back online, everything gets backed
+# up again, as these imports may have non-mvcc ops in them. Ensure this in the
+# unfinalized cluster and in the finalized cluster.
+exec-sql
+BACKUP INTO LATEST IN 'nodelocal://0/cluster/' with revision_history;
+----
+
+
+exec-sql
+BACKUP DATABASE d INTO LATEST IN 'nodelocal://0/database/' with revision_history;
+----
+
+
+query-sql
+SELECT * FROM show_cluster_backup;
+----
+d foo table 1 full
+d foofoo table 2 full
+d foo table 0 incremental
+d foofoo table 0 incremental
+d foo table 2 incremental
+d foofoo table 3 incremental
+
+
+query-sql
+SELECT * FROM show_database_backup;
+----
+d foo table 1 full
+d foofoo table 2 full
+d foo table 0 incremental
+d foofoo table 0 incremental
+d foo table 2 incremental
+d foofoo table 3 incremental
+
+
+upgrade-server version=Start22_2
+----
+
+exec-sql
+BACKUP DATABASE d INTO LATEST IN 'nodelocal://0/database_upgrade/' with revision_history;
+----
+
+query-sql
+SELECT
+  database_name, object_name, object_type, rows, backup_type
+FROM
+  [SHOW BACKUP FROM LATEST IN 'nodelocal://0/database_upgrade/']
+WHERE
+  object_name = 'foo' or object_name = 'foofoo';
+----
+d foo table 1 full
+d foofoo table 2 full
+d foo table 2 incremental
+d foofoo table 3 incremental
+
+
+# A restore on mixed version cluster should always fail on a backup that contains import times
```

Comment:
The backup to database_upgrade won't contain descriptors with import time set right even though its post finalization, because the import was run in a pre-finalized state?